packages/deployer/test/deployer.test.ts

```diff
@@ -332,6 +343,16 @@ describe("Module `deployer`", () => {
               "/portals/abCDefG123456?f=json&token=fake-token",
             portalsSelfResponse
           )
+          .get(
```

Comment:
Needing to make these sort of changes is indicative that the change could be more focused... i.e. did all these functions actually _need_ the basemap? if not, why are we making two additional xhrs in all these places?